src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -207,6 +213,20 @@ public void GetIssues_NoIssuesOnServer_ReturnsEmptyList()
             VerifyServiceGetIssues(Times.Exactly(1));
         }
 
+        [TestMethod]
+        public void GetIssues_UsesExpectedBranch()
```

Comment:
Single new test: make sure the product is asking for and using the server branch.